source/renderer/app/components/splash/SplashNetworkITN.js

```diff
@@ -47,16 +48,18 @@ const messages = defineMessages({
 type Props = {
   onClose: Function,
   openExternalLink: Function,
+  currentTheme: string,
 };
 
+@inject('stores')
```

Comment:
@DeeJayElly please remove this - it is only allowed to inject stores into containers and then pass the values to the components.

Reply:
Its a leftover, i removed it.

Reply:
Its a leftover, i removed it.

Reply:
Are you sure? I still see it on GH? 🤔